processing-for-variant-discovery-gatk4.b37.wgs.inputs.json

```diff
@@ -9,7 +9,6 @@
   "PreProcessingForVariantDiscovery_GATK4.ref_dict": "gs://gatk-legacy-bundles/b37/human_g1k_v37_decoy.dict",
   "PreProcessingForVariantDiscovery_GATK4.ref_fasta": "gs://gatk-legacy-bundles/b37/human_g1k_v37_decoy.fasta",
   "PreProcessingForVariantDiscovery_GATK4.ref_fasta_index": "gs://gatk-legacy-bundles/b37/human_g1k_v37_decoy.fasta.fai",
-  "PreProcessingForVariantDiscovery_GATK4.SamToFastqAndBwaMem.ref_alt": "",
```

Comment:
Why is this empty/removed? both seem wrong.

Reply:
This json uses b37 reference files which does not have an alt file. There is an another hg38 json file which has this variable and path to its alt file.
https://github.com/gatk-workflows/gatk4-data-processing/blob/842f265620fc8cd15ca9f2a713bca631e9935ce9/processing-for-variant-discovery-gatk4.hg38.wgs.inputs.json#L12